src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/utbetalingsoppdrag/ØkonomiUtils.kt

```diff
@@ -92,10 +93,37 @@ object ØkonomiUtils {
         }
     }
 
-    private fun validerOpphørsdato(andelerNyTilkjentYtelse: List<AndelTilkjentYtelse>,
-                                   opphørsDato: LocalDate?) {
-        val nyMinDato = andelerNyTilkjentYtelse.minOfOrNull { it.fraOgMed }
-        if (opphørsDato != null && nyMinDato != null && nyMinDato.isBefore(opphørsDato)) {
+    /**
+     * Skal bruke opphørsdato fra tilkjent ytelse hvis den ikke er den samme som forrige opphørsdato
+     * Hvis ikke så skal den finne finnOpphørsdato, som gjør en diff mellom tidligere og nye andeler
+     */
+    private fun beregnOpphørsdato(forrigeOpphørsdato: LocalDate?,
+                                  nyttOpphørsdato: LocalDate?,
+                                  forrigeAndeler: Set<AndelTilkjentYtelse>,
+                                  oppdaterteAndeler: Set<AndelTilkjentYtelse>): LocalDate? {
+        val opphørsdatoHvisIkkeLikSomForrige = if (forrigeOpphørsdato == nyttOpphørsdato) null else nyttOpphørsdato
```

Comment:
Kan vi risikere at denne går "fremover" i tid? 🤔 Isåfall burde vi ta min(forrigeOpphørsdato, nyOpphørsdato) ?

Reply:
Ser at vi validerer at det ikke skal skje i `validerOpphørsdato`